Source/gg2/Objects/Weapons/Medigun.events/User Event 1.xml

```diff
@@ -16,12 +16,32 @@
         <argument kind="STRING">{
     if(!instance_exists(healTarget) and readyToShoot)
     {
-        var bestTarget, bestTargetDist;
+        var bestTarget, bestTargetDist, testx, testy;
         bestTarget=noone;
-        bestTargetDist=maxHealDistance;
+        bestTargetDist = maxHealDistance;
+        
+        testx = round(owner.x) + cos(degtorad(owner.aimDirection))*owner.aimDistance;
```

Comment:
The trig math was just copied out of the spectator code. I'm a biology major. I haven't taken math in 3 years, so I'm rusty when it comes to things like trigonometry. I kept the rounding in for this reason. If rounding is a bad practice, it's gone. And the trigmath was written by someone else. Wareya, I think. I just used it for the same reason I'd use lengthdir instead of running through the trigonometry. It does what I want it to do in one line.